kratos/python/add_search_strategies_to_python.cpp

```diff
@@ -75,6 +78,81 @@ void CopyRadiusArrayToPython(
 }
 
 /**
+ * @brief Binds a SpatialSearchResultContainer class to Python using pybind11.
+ * @tparam TObjectType The type of object stored in the container.
+ * @param m The pybind11 module to bind the class to.
+ * @param rClassName The name of the class.
+ */
+template<typename TObjectType>
```

Comment:
what object types would be allowed here? GeometricalObject and what else?

Reply:
For now only, but itt will be extended soon